.github/workflows/pythonapp.yml

```diff
@@ -3,6 +3,19 @@ name: cve-bin-tool
 on: [push, pull_request]
 
 jobs:
+  isort:
+    name: Run isort
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
```

Comment:
I had a moment of concern when i saw the 10 minute timeout thinking "how slow is isort?" but it turns out it ran in 13s.  (We could probably set a shorter timeout, but I don't think we really need to.)